include/cvc5/cvc5_kind.h

```diff
@@ -2052,6 +2052,40 @@ enum ENUM(Kind)
    *   - Solver::mkOp(Kind, const std::vector<uint32_t>&) const
    */
   EVALUE(BITVECTOR_TO_NAT),
```

Comment:
Since the NEWS already mention this, the deprecation notice should already be added to the kind.

Reply:
Done